nautobot/core/tests/test_graphql.py

```diff
@@ -1265,15 +1263,15 @@ def test_query_sites_filter(self):
             (f'id: ["{self.site1.pk}", "{self.site2.pk}"]', 2),
             (
                 f'status: "{self.site_statuses[0].slug}"',
-                Site.objects.filter(status__slug__in=[self.site_statuses[0].slug]).count(),
+                Site.objects.filter(status=self.site_statuses[0]).count(),
             ),
             (
                 f'status: ["{self.site_statuses[1].slug}"]',
-                Site.objects.filter(status__slug__in=[self.site_statuses[1].slug]).count(),
+                Site.objects.filter(status=self.site_statuses[1]).count(),
             ),
             (
                 f'status: ["{self.site_statuses[0].slug}", "{self.site_statuses[1].slug}"]',
-                Site.objects.filter(status__slug__in=[self.site_statuses[0].slug, self.site_statuses[1].slug]).count(),
+                Site.objects.filter(status__in=self.site_statuses[:2]).count(),
```

Comment:
This might not work on mysql?

Reply:
It seemed to be working in the CI. May I ask why?

Reply:
MySQL doesn't like filtering on sliced querysets. I think this is OK because `site_statuses` is already being explicitly cast to a list when defined.